zebra-rpc/src/methods.rs

```diff
@@ -9,15 +9,31 @@
 use jsonrpc_core::{self, Result};
 use jsonrpc_derive::rpc;
 
+use crate::server::APP_VERSION;
+use zebra_network::constants::USER_AGENT;
+
 #[rpc(server)]
 /// RPC method signatures.
 pub trait Rpc {
     /// getinfo
     ///
-    /// TODO: explain what the method does
-    ///       link to the zcashd RPC reference
-    ///       list the arguments and fields that lightwalletd uses
-    ///       note any other lightwalletd changes
+    /// Returns software information from the RPC server running Zebra.
+    ///
+    /// zcadhd reference: <https://zcash.github.io/rpc/getinfo.html>
+    ///
+    /// Result:
+    /// {
+    ///      "build": String, // Full application version
+    ///      "subversion", String, // Zebra user agent
+    /// }
+    ///
+    /// Note 1: We only expose 2 fields as they are the only ones needed for
+    /// lightwalletd: <https://github.com/zcash/lightwalletd/blob/v0.4.9/common/common.go#L91-L95>
+    ///
+    /// Note 2: <https://zcash.github.io/rpc/getinfo.html> is outdated so it does not
```

Comment:
Optional: Can you open a ticket at https://github.com/zcash/zcash/issues for this?

The zcashd team have asked us to open tickets for any missing RPC documentation.
(There's no need to do a PR, just let them know which fields are missing.)

Reply:
https://github.com/ZcashFoundation/zebra/pull/3660/commits/dd0b18fa18c20e220c9981b12f5ebc65cafd7859